_data/projects/30SecondsOfC++.yml

```diff
@@ -1,5 +1,7 @@
+---
 name: 30 Seconds of C++
-desc: A collection of STL features (functions/libraries) of C++ which can be learned in 30 seconds or less.
+desc: >-
```

Comment:
Round-tripping strings turned out to be a huge pain in this process, but I think I'm in a happy place with these two choices:

 - setting a line width of 100 characters when emitting the document means most lines are preserved without modification
 - For long `desc` fields, they are emitted with this `>-` which means "Replace newlines with spaces (folded)" and "No newline at end (strip)" - (citation: [yaml-multiline.info](https://yaml-multiline.info/))

Because we don't really support multiple paragraphs on the site itself, preserving strings in this way seems reasonable.